packages/node_modules/@ciscospark/widget-recents/src/components/__snapshots__/UserProfileAvatar.test.js.snap

```diff
@@ -0,0 +1,161 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ProfileAvatar component displays an error page if userProfile is disabled and userProfileMenu is enabled 1`] = `
+<MenuOverlay
```

Comment:
This isn't an error page

Reply:
right. we should not worry about this case.